oneutil/etl/databento.py

```diff
@@ -54,3 +60,130 @@ def get_df_from_s3(
 
     # Return the DataFrame
     return df
+
+
+def get_files_in_s3_folder(
```

Comment:
This function should be moved to `oneutil.etl.aws`. It's not directly related to databento.